src/components/resources/CreateResources.js

```diff
@@ -0,0 +1,285 @@
+import { Form, Button, Card, Alert } from "react-bootstrap"
+import { Link, useHistory } from "react-router-dom"
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { format } from 'date-fns'
+import { useAuth } from "../../contexts/AuthContext"
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+import NavigationBar from '../common/NavigationBar';
+import MuiAlert from '@mui/material/Alert';
+import Grid from '@material-ui/core/Grid';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import TextField from '@material-ui/core/TextField';
+import { makeStyles } from '@material-ui/core/styles';
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const CreateResources = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory()
+
+    //const email = currentUser.email;
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+    const [open, setOpen] = React.useState(false);
+
+
+    React.useEffect(() => {
+        setEmail(currentUser.email);
+        loadUserEmailSearch(currentUser.email);
+    }, []);
+
+    const loadUserEmailSearch = (email) => {
+        callApiGetUserEmailSearch(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetUserEmailSearch = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    const [resourcesTitle, setResourcesTitle] = React.useState('');
+    const [resourcesTitleError, setResourcesTitleError] = React.useState('');
+    const [resourcesTitleErrorText, setResourcesTitleErrorText] = React.useState(''); //ERROR EDITING IN RETURN BRACKETS
+    const handleResourcesTitle = (event) => {
+        setResourcesTitle(event.target.value);
+        setResourcesTitleError(false);
+        setResourcesTitleErrorText('');
+    }
+
+    const [resourcesLink, setResourcesLink] = React.useState('');
+    const [resourcesLinkError, setResourcesLinkError] = React.useState('');
+    const [resourcesLinkErrorText, setResourcesLinkErrorText] = React.useState(''); //ERROR EDITING IN RETURN BRACKETS
+    const handleResourcesLink = (event) => {
+        setResourcesLink(event.target.value);
+        setResourcesLinkError(false);
+        setResourcesLinkErrorText('');
```

Comment:
You could think about implementing logic to ensure the entered text for the resources link is in fact a URL link. But its def not a priority

Reply:
sounds good!

Reply:
this might be a little tricky to do, as you have to check if the string input starts with "https://" or "www."

for simplicity, I think we can leave it but ultimately up to you